lib/feedjira/parser/podcast.rb

```diff
@@ -0,0 +1,82 @@
+require 'feedjira'
+require 'feedjira/parser/media_content'
+require 'feedjira/parser/media_group'
+require 'feedjira/parser/podcast_image'
+require 'feedjira/parser/podcast_item_enclosure'
+require 'feedjira/parser/podcast_item'
+
+module Feedjira
+  module Parser
+    class Podcast
```

Comment:
yes, several libs should be, like prx_access and this, perhaps the itunes categories stuff too